src/core.c

```diff
@@ -323,7 +323,11 @@ void GameInit(void)
 
     VramResetHeapState();
 
+#if !PORTABLE
```

Comment:
Why not just add this to to the `IdentifyFlash` function?

Reply:
Sure!

Reply:
Fixed! ^^